RasterPropMonitor/Core/RPMCEvaluators.cs

```diff
@@ -1601,6 +1601,8 @@ internal NumericVariableEvaluator GetNumericEvaluator(string input, out Variable
                         return double.NaN;
                     };
                 case "TARGETEXISTS":
+                    return (RPMVesselComputer comp) => comp.target == null ? -1d : 1d;
```

Comment:
this is a potentially breaking change; why's it needed?

Reply:
The ASET 40*20 display only showed target information if you were targeting another vessel, whereas the 60*30 display showed target information always. This makes them consistent.

Reply:
I could try to give this variable a new name, and keep the old one as-is